templates/view-program-page.html

```diff
@@ -7,6 +7,11 @@
         <h2>{{_('submitted_header')}}</h2>
         <h3 class="inline-block mt-0">{{_('submission_time')}}: {{program_timestamp}}</h3>
       </div>
+      {% if is_teacher %}
+      <div class="text-center">
+        <button class="green-btn" data-cy="unsubmit-btn" onclick="hedyApp.modal.confirm ('{{_('unsubmit_warning')}}', function () {hedyApp.unsubmit_program ('{{initial_adventure.id}}', {{ loop_index }})})">{{_('unsubmit_program')}}</button>
```

Comment:
```suggestion
        <button class="green-btn" data-cy="unsubmit-btn" onclick="hedyApp.unsubmit_program ('{{initial_adventure.id}}', '{{_('unsubmit_warning')}}'))">{{_('unsubmit_program')}}</button>
```

Either do this and handle the model in the backend, or convert it to htmx as we usually do!